invenio_vocabularies/contrib/names/jsonschemas/names/name-v1.0.0.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "local://names/name-v1.0.0.json",
+  "description": "Names vocabulary.",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "$schema": {
+      "$ref": "local://definitions-v1.0.0.json#/$schema"
+    },
+    "id": {
+      "description": "URI or classification code as identifier - globally unique among all names schemes.",
+      "$ref": "local://definitions-v1.0.0.json#/identifier"
+    },
+    "pid": {
+      "$ref": "local://definitions-v1.0.0.json#/internal-pid"
+    },
+    "scheme": {
+      "description": "Identifier of the name scheme.",
+      "$ref": "local://definitions-v1.0.0.json#/identifier"
+    },
+    "name": {"type": "string"},
+    "given_name": {"type": "string"},
+    "family_name": {"type": "string"},
+    "identifiers": {
+      "description": "Identifiers for the person.",
+      "type": "array",
+      "items": {"$ref": "local://definitions-v1.0.0.json#/identifiers_with_scheme"},
+      "uniqueItems": true
+    },
+    "affiliations": {
+      "description": "Affiliations of the person.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "id": {
+            "$ref": "local://definitions-v1.0.0.json#/identifier"
```

Comment:
Nope, because it is a vocabulary. So while the affiliation itself will have more fields, the "reference" in the name will be:
- `id` if it is an affiliation existing in the DB
- `name` if a custom one

See for example the affiliation in [`rdm-records`](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/records/jsonschemas/records/definitions-v2.0.0.json#L2)

TL;DR: They can have multiple identifiers, yes (in their inner data) but they are referenced by one (the PK in the DB, sort of -  pidstore style)